File: Front/ProEventos-App/src/app/models/Speaker.ts
import { SocialMedia } from "./SocialMedia";
import { Event } from "./Event";

export interface Speaker {
  id: number;
  name: string;
  miniResume: string;
  imageURL: string;
  phone: string;
  email: string;
  socialMedias: SocialMedia[];
  speakerEvents: Event[];
}

File: Back/src/ProEventos.Domain/Speaker.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProEventos.Domain
{
    public class Speaker
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string MiniResume { get; set; }
        public string ImageURL { get; set; }
        public string Phone { get; set; }
        public string Email {get; set;}
        public IEnumerable<SocialMedia> SocialMedias { get; set; }
        public IEnumerable<SpeakerEvent> SpeakerEvents { get; set; }

    }
}
